src/modules/getstates.js

```diff
@@ -0,0 +1,13 @@
+import { addToLocalStorage, tasks } from '../index.js'; //eslint-disable-line
```

Comment:
- [ ] Linters are helpful in ensuring that we have clean code and use the best practices so kindly remove this disable and please try to fix the error instead.

Reply:
Hi, I currently do not have a solution but I am looking for one. However, It might take a while because I am a bit preoccupied at the moment. I will let you know when I have found a solution if you haven't by then.

Reply:
Hi @Nemwel-Boniface can you please check your slack dm